### ISSUE TITLE
Adding #' @NoRd to internal functions

### ISSUE THREAD

**Issue by maelle** [lvaudor/glitter#16]
I.e. functions with a manual page but not exported should probably not get a manual page. E.g. `as_values()`

**Comment by maelle** [comment]
Relatedly, I wonder why e.g. `build_graph_classes()` is exported.

**Comment by lvaudor** [comment]
yes the documention for as_values was for myself but we can put that as comments instead. 
build_graph_classes() should probably disappear from the package anyway (and maybe be part of another package -to maybe come in a maybe quite distant future-, related to the few functions related more specifically to Wikidata classes).
